_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -35,6 +35,7 @@
 #include "mfx_vpx_dec_common.h"
 #include <libmfx_core_vaapi.h>
 
+#define MFX_ERR_INVALID_STREAM ((mfxStatus)UMC::UMC_ERR_INVALID_STREAM)
```

Comment:
I vote for exposure of this status to public API

Reply:
Thank you

Reply:
I don't understand why `VideoDECODEVP9_HW::DecodeFrameHeader` can't return `UMC::Status` and we do need this hack.

Reply:
Just for make code a little bit more readable